app/src/main/java/com/example/simplenavigation/MainActivity.java

```diff
@@ -25,28 +25,56 @@
     private Robot mRobot;
     private final String TAG = MainActivity.class.getSimpleName();
 
+    /**
+     * Function getInt is used to get integer value from EditText views
+     * @param viewId is used to find view we want to get value from
+     * @return is value of view or if view is empty function returns "50" so robot ends up facing upwards towards user
```

Comment:
The syntax for `returns` is:
```java
@return [Description]
```

Instead of what you have, you should write:
```java
@return Value of view or if view is empty function returns "50" so robot ends up facing upwards towards user
```

Reply:
Code changed to suggested line.